scalyr_agent/builtin_monitors/docker_monitor.py

```diff
@@ -508,6 +508,7 @@ def start( self ):
         #create and start the DockerLoggers
         self.__start_docker_logs( self.docker_logs )
         self._logger.log(scalyr_logging.DEBUG_LEVEL_1, "Initialization complete.  Starting docker monitor for Scalyr" )
```

Comment:
I think you can remove this DEBUG_LEVEL_1 log now that the following line prints almost the same message.